src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -1,19 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-linux`
Assembly::LoadWithPartialName Validation Test.Can load native dll
```powershell
Exception calling "LoadNative" with "0" argument(s): "Unable to load shared library '/tmp/tmpepLY8f.tmp/linux-x64/libSkiaSharp' or one of its dependencies. In order to help diagnose loading problems, consider setting the LD_DEBUG environment variable: /tmp/tmpepLY8f.tmp/linux-x64/libSkiaSharp: cannot open shared object file: No such file or directory"
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1: line 51
51:         [TestNativeClass]::LoadNative() | Should -Not -BeNullOrEmpty

```